### ISSUE TITLE
Impossible to move several mails

### ISSUE THREAD

**Issue by tbalay** [kewisch/quickmove-extension#104]
Hello,
Since the last update, I can only move one mail after one.
I was used to select several mails and moved them with quickmove. But now, when I select several mails the "Quick Move" button disappear and the shortcut has no effect.
Thomas

**Comment by tbalay** [comment]
Hello,
On windows it's seems OK.
I have this problem on Ubuntu 22.04.3 LTS with Thundebird 115.2.3 (64 bits) and Quickmove 2.0.1

**Comment by kewisch** [comment]
Do you have the button in the main menubar? This one should work with multiple messages selected.

**Comment by tbalay** [comment]
Yes but it disappear when I select several messages.
1 message :
![image](https://github.com/kewisch/quickmove-extension/assets/145834458/303f3b17-c61f-447d-9b64-e6c32764c4a0)

2 messages :
![image](https://github.com/kewisch/quickmove-extension/assets/145834458/c70f0c99-ae71-4307-a65a-f3090d945747)

**Comment by tbalay** [comment]
Sorry, this is not in the MAIN menu bar.
I added the quickmove button in the MAIN menu bar (it was not), everything's OK.

**Comment by kewisch** [comment]
Great! So it was not in the main menu bar by default? That's odd. I guess I might need a fallback that opens a window in case nothing is visible.

**Comment by kewisch** [comment]
Note also a better solution would be to fix this upstream, which would require https://bugzilla.mozilla.org/show_bug.cgi?id=1847666 fixed.

**Comment by tbalay** [comment]
> Great! So it was not in the main menu bar by default? That's odd. I guess I might need a fallback that opens a window in case nothing is visible.

Exactly, it was not in the main menu bar by default on Ubuntu. But it was on windows.

**Comment by kewisch** [comment]
Can you try and see what happens if you create a new profile and install the add-on? I'd like to find out if this is a recurring issue for everyone that the main toolbar button is not visible.

**Comment by kewisch** [comment]
Ok, I have a fallback for this now which will go into 2.6.5. I'll leave this open to wait for the upstream changes that would allow me to do this directly.

**Comment by AndreKelling** [comment]
I am also experiencing that the "quick move" button is just visible when i have one mail selected.

<img width="631" height="178" alt="Image" src="https://github.com/user-attachments/assets/202d4f41-8edc-4cea-8392-8f47e26a18e5" />

But gladly i now found the Shortcut `[ctrl] + [shift] + [m]` to open the "quick move" menu.
Which works having currently marked multiple mails.

I also do not find a "quick move" button in the main menu of the thunderbird app at all.

----
I am on MacOS 15.6.1 with Thunderbird 140.4.0esr 
Got a quite fresh install of thunderbird here.
